lib/awful/client.lua

```diff
@@ -900,19 +900,18 @@ function client.setwfact(wfact, c)
     t:emit_signal("property::windowfact")
 end
 
---- Increment a client's window factor
+--- Change a client's window factor.
 --
 -- @legacylayout awful.client.incwfact
--- @param add amount to increase the client's window
+-- @tparam number add Amount to increase/decrease the client's window.
+--   Should be between 0.0 and 1.0.
```

Comment:
@Elv13 
But what about making it idem-potent, so that calling it with -0.1 and 0.1 (multiple times) results in the same layout?
What is the new API for this?